src/rgw/rgw_rest_swift.cc

```diff
@@ -1404,6 +1406,83 @@ RGWOp *RGWHandler_REST_Service_SWIFT::op_delete()
   return NULL;
 }
 
+static bool is_web_mode(const req_state * const s) {
+  const boost::string_ref webmode = s->info.env->get("HTTP_X_WEB_MODE", "");
+  return webmode == "true";
+}
+
+static bool is_website_req(const req_state * const s)
```

Comment:
function name is misleading, maybe something like `can_be_website_req()` or something similar?